logins-sql/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "logins-sql"
+version = "0.1.0"
+authors = ["Thom Chiovoloni <tchiovoloni@mozilla.com>"]
+
+[dependencies]
+sync15-adapter = { path = "../sync15-adapter" }
+serde = "1.0.75"
+serde_derive = "1.0.75"
+serde_json = "1.0.26"
+log = "0.4.4"
+lazy_static = "1.1.0"
+more-asserts = "0.2.1"
```

Comment:
I'm the author of that crate, it helps when debugging issues since it shows the actual values that failed. e.g. `assert_lt!(a, b)` will show both `a` and `b` whereas `assert!` won't unless you provide the formatting args.

IMO this has value, but I also don't mind taking it out if you think it would be hard to land in m-c with it.